platforms/shared/configuration/roles/setup/certmanager/meta/main.yml

```diff
@@ -0,0 +1,52 @@
+galaxy_info:
```

Comment:
Please fix this file to include only the relevant vars and their corresponding comments.

Reply:
removed meta/main.yml